modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/util/DLProcessorRegistryImpl.java

```diff
@@ -275,6 +276,13 @@ private FileVersion _getLatestFileVersion(
 
 			return null;
 		}
+		catch (NoSuchFileVersionException noSuchFileVersionException) {
```

Comment:
@yuhai Why you add NoSuchFileVersionException checking instead of modify NoSuchFileEntryException to NoSuchFileVersionException, any reason you need NoSuchFileEntryException